source-map-uploader-plugin.js

```diff
@@ -175,11 +175,6 @@ class BugsnagSourceMapUploaderPlugin {
       }
     }
 
-    // Boolean flags
-    if (opts.overwrite) {
```

Comment:
Is this no longer required? It might be worth asserting that boolean types make it through in the tests for both upload and build?